modules/core/src/main/scala/fs2/kafka/security/package.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018-2021 OVO Energy Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package fs2.kafka
+
+import java.nio.charset.StandardCharsets
+import java.security.MessageDigest
+import scala.annotation.tailrec
+
+package object security {
```

Comment:
Maybe we could consider moving it to a separate module?

Reply:
Thanks @LMnet, I did wonder about that. It just seemed like it wasn't that separate from the core. Happy to move it out if that's what people want 👍

Reply:
Actually, this code is blatantly copied from `Secret` in ciris. Perhaps it should be its own library, because I'm sure I've seen code very similar to this in a few different places.

Reply:
I initially agreed with @LMnet, but I think it's better to keep it in the same module so we can have the `withCredentials` helpers on the settings classes.